src/locales/en-gb.ts

```diff
@@ -1,9 +1,10 @@
+import Locale from "./locale";
 import { enGB } from "./date-fns-locales";
 
-const isSingular = (count) =>
+const isSingular = (count: string | number): boolean =>
   (typeof count === "string" ? parseInt(count) : count) === 1;
 
-export default {
+const EnGb: Locale = {
```

Comment:
I think that should be `enGB` to be consistent with our naming convention. You could import `enGB` dateFnsLocale as an alias to avoid name conflict.